hw14.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <link rel="stylesheet" href="./styles.css">
+</head>
+<body>
+    
+    <span class="container">
+       
+        <span class="Trigger top1"><div class="tooltip top2">I'm the top tooltip</div>Trigger top</span>
+        <span class="Trigger right1"><div class="tooltip right2">I'm the right tooltip</div>Trigger right</span>
```

Comment:
is there a reason to have both "right1" and "right2" classes?
I bet the one "right" in trigger is enough. for the tooltip you can use parent's class "right"

The same for all emenents